design-system-docs/config/webpack.defaults.js

```diff
@@ -17,6 +17,7 @@ const { WebpackManifestPlugin } = require('webpack-manifest-plugin');
 
 const entry = {
   main: path.resolve(rootDir, 'frontend', 'javascript', 'index.js'),
+  theme: path.resolve(rootDir, 'frontend', 'javascript', 'theme.js'),
```

Comment:
The way these entrypoints work is super confusing. Would love to migrate away from webpack to something simpler eventually.